### PR TITLE
006 system forget password logic

### PR DESCRIPTION
**Pull request by eyalgolan**
fixed previous issues

### CODE REVIEW

File: src/server/server.js [hunk 1]
```diff
@@ -43,7 +43,8 @@ app.use("/users", routers.userRouter);
 // Index page routing
 app.use("/index", routers.indexRouter);
 
-const port = process.env.PORT || 5000;
+const defaultPort = 5000;
```
Comment (teddy4445) [inline src/server/server.js:46]: DEFAULT_PORT
check out @arielfenster 's design coding style file

File: src/server/src/routes/users.js [hunk 2]
```diff
@@ -10,58 +10,63 @@ dotenv.config();
 
 const router = express.Router();
 
+const mg = mailgun({
```
Comment (teddy4445) [inline src/server/src/routes/users.js:13]: use a more explainable name as everybody knows what "mg" stands for

File: src/server/src/routes/users.js [hunk 3]
```diff
@@ -10,58 +10,63 @@ dotenv.config();
...
+  apiKey: process.env.MAILGUN_APIKEY,
+  domain: process.env.MAILGUN_DOMAIN,
+});
+
+// Hash passwords
+const hashPassword = async (password) => {
+  const salt = await bcrypt.genSalt(10);
```
Comment (teddy4445) [inline src/server/src/routes/users.js:20]: magic number...

Comment (teddy4445) [reply src/server/src/routes/users.js:20]: You can leave it and write a comment why "10" is a good choice

File: src/server/src/routes/users.js [hunk 4]
```diff
@@ -74,43 +79,33 @@ router.post("/login", async (req, res) => {
   if (error) return res.status(400).send(error.details[0].message);
```
Comment (teddy4445) [inline src/server/src/routes/users.js:79]: magic number

**Review by teddy4445 — APPROVED** [review]
Small coding fixes - approved

File: src/server/server.js (hunk 1 above)
Comment (eyalgolan) [reply src/server/server.js:46]: fixed

File: src/server/src/routes/users.js (hunk 2 above)
Comment (eyalgolan) [reply src/server/src/routes/users.js:13]: fixed

File: src/server/src/routes/users.js (hunk 3 above)
Comment (eyalgolan) [reply src/server/src/routes/users.js:20]: fixed

File: src/server/src/routes/users.js (hunk 4 above)
Comment (eyalgolan) [reply src/server/src/routes/users.js:79]: fixed

**Pull request merged**